packages/core/js-client/src/index.ts

```diff
@@ -30,36 +26,23 @@ import {
 import { callAquaFunction } from "./compilerSupport/callFunction.js";
 import { registerService } from "./compilerSupport/registerService.js";
 import { MarineBackgroundRunner } from "./marine/worker/index.js";
-import { doRegisterNodeUtils } from "./services/NodeUtils.js";
-
-import { fetchResource } from "#fetcher";
+import versions from "./versions.js";
 
 const DEFAULT_CDN_URL = "https://unpkg.com";
 
-const isNode =
-  // process.release is undefined in browser env
-  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-  typeof process !== "undefined" && process.release?.name === "node";
+const getVersionedPackage = (pkg: keyof typeof versions) => {
+  return `${pkg}@${versions[pkg]}`;
+};
```

Comment:
I would suggest to do the concatination inside browser fetchResource so there is no need to do the parsing in  getWorker and fetchResource for node and return {package: string, version: string} from this function

Reply:
this concatenation could not lie in code with fetchResources, versions not available there.
Though i agree about returning object instead of string

Reply:
I mean only the concatenation part can be done there. The versions file can be used here as it is right now. I do not suggest using versions file there

Reply:
Ok, got it. Done